### ISSUE TITLE
Presence subscription example

### ISSUE THREAD

**Issue by srcnysf** [ably/ably-flutter#53]
Hi guys, I have been using the SDK for a while. First off all I want to thank you thant you made and keep making great job here and I didn't face with any warnings or errors yet. 

However when I tried to use Presence I didn't made that happen bcs the methods and parameters are not similar with doc in  [ably website.]( https://www.ably.io/documentation/realtime/presence)
So can you please help me with that? 

@zoechi @tiholic @QuintinWillison

**Comment by tiholic** [comment]
@srcnysf Channel presence is not yet handled, and is expected to be soon.
We'll be sharing more details once we have any update this.

**Comment by srcnysf** [comment]
any updates for realtime presence? @QuintinWillison @tiholic

**Comment by QuintinWillison** [comment]
Thanks for your tenacity and patience! 😁 

We've had a lot on over the last couple of months and this hasn't been top priority. I'm hoping we'll pick up the pace this in the next couple of weeks. I know @tiholic has already done some work on this but some of that is held up on me finding the time to review that work.

Sorry for the delay. Please bear with us.
